python-sdk/tests/sql/operators/test_dataframe.py

```diff
@@ -334,3 +335,24 @@ def sample_df_1(**kwargs):
 
     task = sample_df_1(output_table=output_table)
     assert task.operator.outlets == []
+
+
+def test_col_capitalization_default_dataframe(sample_dag):
+    """Test that capitalization of col remains the same"""
+
+    assert settings.COLUMN_CAPITALIZATION == "original"
```

Comment:
You can use pytest parameters to make this test apply to all three known settings (and maybe even an invalid setting?)